packages/network/src/chart/Link.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { linkStyleShape } from '../utils/propShapes';
+
+const proptypes = {
+  linkStyles: linkStyleShape,
+  link: PropTypes.object.isRequired,
```

Comment:
this is where I would define a `linkShape`, and you can export it in the package